extensions/git/src/git.ts

```diff
@@ -686,7 +686,7 @@ export class Git {
 	}
 
 	async addSafeDirectory(repositoryPath: string): Promise<void> {
-		await this.exec(repositoryPath, ['config', '--global', '--add', 'safe.directory', repositoryPath]);
+		await this.exec('', ['config', '--global', '--add', 'safe.directory', repositoryPath]);
```

Comment:
Would it make more sense to use `os.homedir()` here, instead of `''`? I noticed the same pattern here: https://github.com/microsoft/vscode/blob/6d40104789d03d41b8866a1e57847dae14c5cf0d/extensions/git/src/git.ts#L676